01week/rockPaperScissors.js

```diff
@@ -11,9 +11,86 @@ const rl = readline.createInterface({
 function rockPaperScissors(hand1, hand2) {
 
   // Write code here
+//hand 1 enters their weapon, hand 2 enters their weapon - This is done
 
+if(typeof(hand1)=='string' && typeof(hand2)=='string'){
+} else {
+console.log('Invalid entry, please choose rock, paper, or scissors')
+}
+//making sure the hands entered are strings
+
+
+if (hand1 != 'rock' && hand1 != 'paper' && hand1 != 'scissors'){
+  console.log('Invalid entry from player 1, please choose rock, paper, or scissors')
+}
+//making sure hand1 is rock, paper, or scissors
+
+if (hand2 != 'rock' && hand2 != 'paper' && hand2 != 'scissors'){
+  console.log('Invalid entry from player 2, please choose rock, paper, or scissors')
+}
+//making sure hand2 is rock, paper, or scissors
+
+if(hand1==hand2){
+  console.log("It's a tie!")
+}
+//checking for the tie first to narrow the cases
+
+if (hand1 == 'rock' && hand2 == "paper"){
+  console.log("Hand two wins!")
+}
+
+if (hand1 == 'rock' && hand2 == "scissors"){
+  console.log("Hand one wins!")
```

Comment:
Console logging a string will not end the function, in addition, the prompt function console logs the return of this function. Return this string